### PR TITLE
Update composer.md guide to capture locked farmOS version

### PR DESCRIPTION
**Pull request by symbioquine**
Having the version appear in the status report can be really helpful for debugging and improves the overall polish of these instructions IMHO :)

### CODE REVIEW

File: docs/hosting/composer.md [hunk 1]
```diff
@@ -213,6 +213,9 @@ Example `Dockerfile`:
 # Inherit from the upsteam farmOS 2.x image.
 FROM farmos/farmos:2.x
 
+# Install `jq` to help in extracting the farmOS version below
```
Comment (mstenta) [inline docs/hosting/composer.md]: End comment with a period.

**Review by mstenta — CHANGES_REQUESTED** [review]
LGTM! Just one minor punctuation nit. :-)

**Comment by symbioquine** [comment]
Force push added the requested punctuation, added a changelog entry, and modified the commit message to include the PR #.

**Comment by mstenta** [comment]
Merged - thanks @symbioquine!

**Comment by paul121** [comment]
It seems farmOS.org has not been updated with these changes: https://farmos.org/hosting/composer/#docker

I think the github action ran, but maybe the netlify build glitched? https://github.com/farmOS/farmOS/actions/runs/5454632918/jobs/9925013169

**Comment by wotnak** [comment]
Looks like the Node.js version used for website build is too old. https://app.netlify.com/sites/gracious-brattain-bdd606/deploys/64a40f042600ba00081abaa6#L93
It can be configured in Netlify dashboard or using ntelify.toml file like I've done it in https://github.com/farmOS/farmOS.org/pull/79/files#diff-ab8f79b68b7adff7a07db953bf453f3c5aa6ade98d2b1b67d8432b36392489edR3.

**Comment by mstenta** [comment]
FYI the issue was resolved by https://github.com/farmOS/farmOS.org/pull/79

**Pull request merged**